source/whats-new.txt

```diff
@@ -48,31 +79,74 @@ The following changes affect future versions of the driver:
                       new Document("$collStats",
                           new Document("storageStats", new Document()))));
 
-  To determine whether the collection is a capped collection, access the value
-  of the ``storageStats.capped`` field returned by this example aggregation.
+  To determine whether a collection is a capped collection, access the value
+  of the ``storageStats.capped`` field returned by the preceding aggregation.
```

Comment:
Nit: I think it would still be good to include "example" here to point to what aggregation you mean. 
```suggestion
  of the ``storageStats.capped`` field returned by the preceding example aggregation.
```